### PR TITLE
Fix LayerNorm ONNX export

### PR DESCRIPTION
**Pull request by nzmora-nvidia**
When exporting LayerNorm make sure that the weights and bias inputs have the same type as the LN input.
Also:
 * Add a regression test.
 * Add environment variable to override directory of generated test artifacts

### CODE REVIEW

File: tests/pytorch/test_onnx_export.py [hunk 1]
```diff
@@ -50,7 +50,9 @@
     from polygraphy.comparator import RunResults
 
 # The directory where generated ONNX test models are stored.
-TEST_ARTIFACTS_DIR = os.path.join(tempfile.gettempdir(), "./gen_onnx_models")
+TE_TEST_ARTIFACTS_DIR = os.environ.get('TE_TEST_ARTIFACTS_DIR')
```
Comment (timmoon10) [inline tests/pytorch/test_onnx_export.py:53]: TE environment variables should be prefixed with `NVTE_`:
```suggestion
NVTE_TEST_ARTIFACTS_DIR = os.environ.get('NVTE_TEST_ARTIFACTS_DIR')
```

**Review by timmoon10 — CHANGES_REQUESTED** [review]
Overall LGTM. We just want an environment variable to match our naming convention.

File: tests/pytorch/test_onnx_export.py (hunk 1 above)
Comment (ksivaman) [reply tests/pytorch/test_onnx_export.py]: All instances updated

**Review by timmoon10 — APPROVED** [review]

**Review by ksivaman — APPROVED** [review]
Thanks @nzmora-nvidia!

**Pull request merged**